src/inmanta/data/__init__.py

```diff
@@ -1681,7 +2042,6 @@ def get_composed_filter_with_query_types(
             query_type, value = value_with_query_type
             filter_statement: str
             filter_values: List[object]
-            cls.validate_field_name(key)
```

Comment:
Can I do this?

Reply:
I don't really get the question.

Reply:
The issue here is that I remove the check that validates if the field name is correct, assuming it has been validated before,